api/rpc_handler.go

```diff
@@ -0,0 +1,63 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/topfreegames/pitaya-admin/usecases"
+
+	"github.com/topfreegames/pitaya"
+)
+
+// RPCHandler handler
+type RPCHandler struct {
+	App *App
+}
+
+// NewRPCHandler creates a new handler
+func NewRPCHandler(a *App) *RPCHandler {
+	m := &RPCHandler{App: a}
+	return m
+}
+
+// RPCRequest struct
+type RPCRequest struct {
```

Comment:
why is this duplicated here? you can use from the definition from the other package